numba/targets/externals.py

```diff
@@ -150,7 +156,46 @@ def _do_install(self, context):
             # (under Windows, different versions of the C runtime can
             #  be loaded at the same time, for example msvcrt100 by
             #  CPython and msvcrt120 by LLVM)
-            ll.add_symbol(fname, c_helpers[fname])
+            if need_kb982107 and fname.startswith('fmod'):
+                ll.add_symbol(fname, c_helpers['fixed_' + fname])
+            else:
+                ll.add_symbol(fname, c_helpers[fname])
+
+        if need_kb982107:
+            set_fnclex(context, c_helpers)
+
+
+def set_fnclex(context, c_helpers):
+    """
+    Install fnclex before fmod calls.
+    Workaround for https://support.microsoft.com/en-us/kb/982107
+    """
+    ptr_set_fnclex = c_helpers['set_fnclex']
+    fn = ctypes.CFUNCTYPE(None, ctypes.c_void_p)(ptr_set_fnclex)
+
+    library = compile_fnclex(context)
```

Comment:
I wonder what happens when the library goes out of scope. Presumably LLVM currently doesn't release allocated code memory for the module, but perhaps it will do in the future?